pkg/serverless/remoteconfig/remoteconfig.go

```diff
@@ -0,0 +1,103 @@
+		traceAgentEnv := configUtils.GetTraceAgentDefaultEnv(config)
+
+		options := []remoteconfig.Option{
+			remoteconfig.WithAPIKey(apiKey),
+			remoteconfig.WithTraceAgentEnv(traceAgentEnv),
+			remoteconfig.WithConfigRootOverride(config.GetString("site"), config.GetString("remote_configuration.config_root")),
+			remoteconfig.WithDirectorRootOverride(config.GetString("site"), config.GetString("remote_configuration.director_root")),
+			remoteconfig.WithRcKey(config.GetString("remote_configuration.key")),
+		}
+
+		if config.IsSet("remote_configuration.refresh_interval") {
+			options = append(options, remoteconfig.WithRefreshInterval(config.GetDuration("remote_configuration.refresh_interval"), "remote_configuration.refresh_interval"))
+		}
+		if config.IsSet("remote_configuration.max_backoff_interval") {
+			options = append(options, remoteconfig.WithMaxBackoffInterval(config.GetDuration("remote_configuration.max_backoff_interval"), "remote_configuration.max_backoff_interval"))
+		}
+		if config.IsSet("remote_configuration.clients.ttl_seconds") {
+			options = append(options, remoteconfig.WithClientTTL(config.GetDuration("remote_configuration.clients.ttl_seconds"), "remote_configuration.clients.ttl_seconds"))
+		}
+		if config.IsSet("remote_configuration.clients.cache_bypass_limit") {
+			options = append(options, remoteconfig.WithClientCacheBypassLimit(config.GetInt("remote_configuration.clients.cache_bypass_limit"), "remote_configuration.clients.cache_bypass_limit"))
+		}
+		tagsGetter := func() []string {
+			arn, parseErr := arn.Parse(functionARN)
+			if parseErr != nil {
+				log.Debugf("unable to parse function ARN: %v, not setting tags", functionARN)
+				return []string{}
+			}
+			tags := []string{fmt.Sprintf("aws_account_id:%s", arn.AccountID), fmt.Sprintf("region:%s", arn.Region)}
+			log.Debugf("setting RC service tags: %v", tags)
+			return tags
+		}
+		commonOpts := telemetry.Options{NoDoubleUnderscoreSep: true}
+		telemetryReporter := &rctelemetryreporterimpl.DdRcTelemetryReporter{
+			BypassRateLimitCounter: telemetry.NewCounterWithOpts(
+				"remoteconfig",
+				"cache_bypass_ratelimiter_skip",
+				[]string{},
+				"Number of Remote Configuration cache bypass requests skipped by rate limiting.",
+				commonOpts,
+			),
+			BypassTimeoutCounter: telemetry.NewCounterWithOpts(
+				"remoteconfig",
+				"cache_bypass_timeout",
+				[]string{},
+				"Number of Remote Configuration cache bypass requests that timeout.",
+				commonOpts,
+			),
+		}
+
+		configService, err := remoteconfig.NewService(
+			config,
+			"Remote Config",
+			baseRawURL,
+			functionARN,
+			tagsGetter,
+			telemetryReporter,
+			version.AgentVersion,
+			options...,
+		)
+		if err != nil {
+			log.Errorf("unable to create remote config service: %v", err)
+			return nil
+		}
+		configService.Start()
+		return configService
+	}
+	log.Debug("Remote configuration configuration is disabled, did not create service")
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
	log.Debug("Remote Config is disabled, did not create service")
```

Reply:
nice catch, thank you!